INSTALL.rst

```diff
@@ -29,14 +29,14 @@ Running services locally
 Prerequisites
 =============
 
-HEPData runs with Python 3.7, 3.8 or 3.9. It also uses several services, which you will need to install before running HEPData:
+HEPData runs with Python 3.7, 3.8 or 3.9. It also uses several services, which you will need to install before running HEPData.
+These services can be installed using the relevant package manager for your system,
+for example, using ``yum`` or ``apt-get`` for Linux or ``brew`` for macOS:
```

Comment:
Need to insert a blank line here to avoid an error message: `../INSTALL.rst:35: ERROR: Unexpected indentation.`